challenges/desafio22.js

```diff
@@ -0,0 +1,14 @@
+db.voos.find(
+  {
+    $or: [
+      {"empresa.nome": "DELTA AIRLINES"},
+      {"empresa.nome": "AMERICAN AIRLINES"}
+    ],
```

Comment:
Nessa query vc ja usou o $and implicito, TOP! Eu mudaria talvez um pouco o $or pra nao ter que repetir a chave "empresa.nome" duas vezes:
```suggestion
  "empresa.nome": { $in: ["DELTA AIRLINES", "AMERICAN AIRLINES"] },
```